pkg/diagnostics/network/run_pod.go

```diff
@@ -143,7 +142,7 @@ func (d *NetworkDiagnostic) runNetworkDiagnostic() {
 	}
 
 	// Collection Phase: Run network diagnostic pod on all valid nodes
-	command = []string{"chroot", util.NetworkDiagContainerMountPath, "sleep", "1000"}
+	command = "sleep 1000"
```

Comment:
Network diagnostics has 2 phases: Testing and Collection phase.
In Testing phase, various network related diagnostics are performed on every node and results are stored on the respective nodes. In Collection phase, results from each node are moved to the user machine where the CLI cmd is executed.
This magic sleep is needed for collection phase that will block the network diagnostic pod on every node so that results can be moved to the user machine.